adm/admin_account.php

```diff
@@ -8,15 +8,14 @@
 *
 */
 
-if (!defined('IN_ICYPHOENIX')) define('IN_ICYPHOENIX', true);
-
-if(!empty($setmodules))
+if(defined('IN_ICYPHOENIX') && !empty($setmodules))
```

Comment:
I will accept this, but at the same time I will need to understand which is the advantage to have it in this form... I may revert it if I'm not able to double check all single pages.